noxfile.py

```diff
@@ -50,9 +50,25 @@ def build_and_run_test(session: nox.Session, folders: List, extras: Optional[str
 def test_with_only_required_packages(session: nox.Session):
     """Tests that only depends for required libraries"""
     # Logger
+    # Metrics - Amazon CloudWatch EMF
+    # Metrics - Base provider
     build_and_run_test(
         session,
         folders=[
-            f"{PREFIX_TESTS_FUNCTIONAL}/logger/",
+            f"{PREFIX_TESTS_FUNCTIONAL}/logger/required_dependencies/",
+            f"{PREFIX_TESTS_FUNCTIONAL}/metrics/required_dependencies/",
```

Comment:
see how beautiful it is now - both in Nox and on folder structure <3 no ambiguity now!